System-Testing-Deployments-and-Upgrades/User-Authorization/index.rst

```diff
@@ -0,0 +1,162 @@
+
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Wouter van Reeven*, *Rob Bovill*
+
+.. _User-Authorization:
+
+##################
+User Authorization
+##################
+
+The Rubin Observatory Control System has an authorization system designed to help manage and coordinate user interaction with the system.
+
+By default, the system is designed to be operating under autonomous CSC control.
```

Comment:
system = observatory? 
Consider:
By default, the observatory is designed to perform regular survey observations under autonomous CSC control.